lib/pageflow/chart/scraper.rb

```diff
@@ -103,6 +126,18 @@ def script_src_tags_in_head
         document.css('head script[src]')
       end
 
+      def filtered_script_tags_in_body
+        script_src_tags_in_body.reject do |tag|
+          options.fetch(:body_script_blacklist, []).any? do |regexp|
+            tag[:src] =~ regexp
+          end
+        end
+      end
+
+      def script_src_tags_in_body
+        document.css('body script[src]')
+      end
+
```

Comment:
This looks like a good opportunity for refactoring to remove the duplication between these `_body` and `_head` methods.